kprogresshud/build.gradle

```diff
@@ -37,6 +37,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
+    compile 'org.jetbrains:annotations-java5:15.0'
```

Comment:
Using this type of annotation is generally good for the IDE to spot bugs and output some warnings. 
But in this case I think it not so important to add a lib just for a single @Nullable. IMHO.